base/sort.jl

```diff
@@ -689,12 +693,14 @@ function sortperm(v::AbstractVector;
     for (i,ind) in zip(eachindex(p), indices(v, 1))
         p[i] = ind
     end
-    sort!(p, alg, Perm(ordr,v))
+    sort!(p, alg, Perm(order, v))
 end
 
 
 """
-    sortperm!(ix, v; alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity, rev::Bool=false, order::Ordering=Forward, initialized::Bool=false)
+    sortperm!(ix, v;
+              alg::Algorithm=DEFAULT_UNSTABLE, lt=isless, by=identity,
```

Comment:
Ah. Maybe it has improved, but now it looks just like it should -- and quite nicer than long lines, which are hidden and require horizontal scrolling to be read.